ouster_ros/config/ouster.yaml

```diff
@@ -0,0 +1,20 @@
+  # hostname or IP in dotted decimal form of the sensor
```

Comment:
Is there a specific reason for moving away from the current style of the launch and no config files?

Reply:
Labeling this as Q1